demos/sample_azure_iot_adu/sample_azure_iot_adu.c

```diff
@@ -577,9 +579,9 @@ static AzureIoTResult_t prvEnableImageAndResetDevice()
      * through pucResultDetails.
      */
     xUpdateResults.lResultCode = 0;
-    xUpdateResults.lExtendedResultCode = 0;
-    xUpdateResults.pucResultDetails = NULL;
-    xUpdateResults.ulResultDetailsLength = 0;
+    xUpdateResults.lExtendedResultCode = 1234;
```

Comment:
does this number have some significance? If I'm a first time user I might think this means something. So maybe a macro titled `sampleaduEXAMPLE_EXTENDED_RESULT` or something to make it explicit this is just made up?

Reply:
That sounds like a good idea. I'll move it to be a macro.